src/components/NcAppSettingsSection/NcAppSettingsSection.vue

```diff
@@ -66,7 +66,6 @@ export default {
 .app-settings-section {
 	margin-bottom: 80px;
 	&__name {
-		font-size: 20px;
```

Comment:
We do not have font sizes on Nextcloud 28 so we need a meaningful value also for 28, otherwise it looks like text.

Reply:
Do you know any trick to achieve this?

Reply:
Just enforce the size we use in Nextcloud 30 :sweat_smile:

Reply:
Should be `26px` on Nextcloud 30, so I guess we can hard code it here